xcode/Subconscious/SubconsciousTests/Tests_DataService.swift

```diff
@@ -133,7 +133,8 @@ final class Tests_DataService: XCTestCase {
             let noosphere = NoosphereService(
                 globalStorageURL: globalStorageURL,
                 sphereStorageURL: sphereStorageURL,
-                gatewayURL: GatewayURL("http://unavailable-gateway.fakewebsite")
+                gatewayURL: GatewayURL("http://unavailable-gateway.fakewebsite"),
+                errorLoggingService: SentryIntegration()
```

Comment:
Will this result in sentry errors being logged during testing? I believe sentry is globally disabled during testing but maybe we should stub out a dummy integration here?

Reply:
Good catch! Yes, we should.